python/ray/serve/tests/test_pipeline_dag.py

```diff
@@ -371,6 +409,35 @@ def test_single_functional_node_base_case(serve_instance):
     assert requests.get("http://127.0.0.1:8000/").text == "1"
 
 
+class TestServeBuild:
```

Comment:
it will be easier if we split this into separate file given it's asserting slightly different things, and `test_pipeline_dag` is a file that we put too much stuff in one place during branch cut sprint that we need to split soon as well ..

Reply:
I moved the tests to `test_application.py`.